### PR TITLE
Campaign Classic 3.x (Swift) Update read me instruction

### PR DESCRIPTION
**Pull request by PravinPK**
## Description

Update the missed out CampaignClassic 3.x install instruction from gitbook (PR)

### CODE REVIEW

File: src/pages/documentation/adobe-campaign-classic/tabs/index.md [hunk 1]
```diff
@@ -35,21 +60,22 @@ pod 'ACPCore'
 
 2. In the Xcode project, import the Mobile Core and Campaign Classic extensions:
 
-**Objective-C**
+#### Swift
+
+```swift
+   import ACPCore
+   import ACPCampaignClassic
+   import ACPLifecycle
```
[the remark this answers is not in the document]
Comment (swarna04) [reply src/pages/documentation/adobe-campaign-classic/tabs/index.md:68]: Remove `pod 'ACPLifecycle'` on L57 as well.

File: src/pages/documentation/adobe-campaign-classic/tabs/index.md [hunk 2]
```diff
@@ -78,7 +104,31 @@ public class CampaignClassicTestApp extends Application {
 
 <Variant platform="ios-aep" task="register" repeat="1"/>
```
Comment (swarna04) [inline src/pages/documentation/adobe-campaign-classic/tabs/index.md:105]: Fix the `repeat` count here too.

**Review by swarna04 — CHANGES_REQUESTED** [review]
@PravinPK I've provided feedback here.

File: src/pages/documentation/adobe-campaign-classic/tabs/index.md [hunk 3]
```diff
@@ -78,7 +104,31 @@ public class CampaignClassicTestApp extends Application {
...
-Adobe Campaign Classic has not yet been released as an AEP 3.x Swift extension. Please reach out to your Adobe customer account manager if you have any questions or would like to express interest in the AEP 3.x Campaign Classic extension.
+In your app's `application:didFinishLaunchingWithOptions:` method, register the Campaign Classic extension:
+
+#### Swift
+
+```swift
+// AppDelegate.swift
+func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+  MobileCore.registerExtensions([CampaignClassic.self], {
+    MobileCore.configureWith(appId: "APP-ID")
```
[the remark this answers is not in the document]
Comment (PravinPK) [reply src/pages/documentation/adobe-campaign-classic/tabs/index.md:115]: Good point. 
But we might need to replace this in all our extensions to be uniform. Should I create a task for it ?

**Review by hbhandal80 — COMMENTED** [review]
Looks good to me :)

File: src/pages/documentation/adobe-campaign-classic/tabs/index.md [hunk 4]
```diff
@@ -17,9 +17,34 @@ import com.adobe.marketing.mobile.CampaignClassic;
 import com.adobe.marketing.mobile.Lifecycle;
 ```
 
-<Variant platform="ios-aep" task="add" repeat="1"/>
+<Variant platform="ios-aep" task="add" repeat="8"/>
```
Comment (emdobrin) [inline src/pages/documentation/adobe-campaign-classic/tabs/index.md:20]: does this tab show the objectivec example or should this be 9?

**Pull request merged**